src/Command/Core/InitCommand.php

```diff
@@ -134,19 +160,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $params = [
-            'adapter' => $repositoryManager,
+            'repo_adapter' => $repositoryManager,
             'issue_tracker' => $issueTracker,
+            'repo_org' => $org,
+            'repo_name' => $repo,
+            'issue_org' => $issueOrg,
```

Comment:
either it is issue_repo_org or the one below is just issue_name, or things are confusing :older_man: